src/fable/Fable.Compiler/FSharp2Fable.Util.fs

```diff
@@ -433,6 +434,14 @@ module Types =
         // The F# compiler "untuples" the args in methods
         | args -> List.concat args |> List.map (fun x -> makeType com Context.Empty x.Type)            
 
+    and makeFullType com (args: IList<IList<FSharpParameter>>) returnType = 
```

Comment:
The name of this function may be a bit confusing as there is already `makeType` which doesn't produce partial types. I'd suggest something like `makeTypeFromParams` (there's also `makeTypeFromDef`).